app/src/main/res/layout/item_stock.xml

```diff
@@ -41,71 +39,115 @@
-        app:layout_constraintEnd_toEndOf="parent"
-        android:layout_marginTop="8dp"
-        />
+        app:layout_constraintStart_toStartOf="@+id/rthText"
+        app:layout_constraintTop_toBottomOf="@+id/changeValue"
+        tools:text="+9999.99%" />
 
     <com.robinhood.ticker.TickerView
         android:id="@+id/changeValue"
-        android:layout_width="match_parent"
+        style="@style/Widget.StocksWidget.TextView.BodyMedium"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical|end"
+        android:maxLines="1"
+        android:textAppearance="@style/TextAppearance.StocksWidget.BodyMedium"
+        app:layout_constraintStart_toStartOf="@+id/rthText"
+        app:layout_constraintTop_toBottomOf="@+id/totalValue"
+        tools:text="+12,345.00" />
+
+    <com.robinhood.ticker.TickerView
+        android:id="@+id/othValue"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical|start"
+        android:maxLines="1"
+        android:textAppearance="@style/TextAppearance.StocksWidget.TickerBold"
+        app:layout_constraintStart_toStartOf="@+id/othText"
+        app:layout_constraintTop_toBottomOf="@+id/othText"
+        tools:text="123,456.78" />
+
+    <com.robinhood.ticker.TickerView
+        android:id="@+id/othChangePercent"
+        style="@style/Widget.StocksWidget.TextView.BodyMedium"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="4dp"
         android:gravity="center_vertical|end"
         android:maxLines="1"
+        android:textAppearance="@style/TextAppearance.StocksWidget.BodyMedium"
+        app:layout_constraintStart_toStartOf="@+id/othChangeValue"
+        app:layout_constraintTop_toBottomOf="@+id/othChangeValue"
+        tools:text="+1111.11%" />
+
+    <com.robinhood.ticker.TickerView
+        android:id="@+id/othChangeValue"
         style="@style/Widget.StocksWidget.TextView.BodyMedium"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical|end"
+        android:maxLines="1"
         android:textAppearance="@style/TextAppearance.StocksWidget.BodyMedium"
-        tools:text="15.02345"
+        app:layout_constraintStart_toStartOf="@+id/othText"
+        app:layout_constraintTop_toBottomOf="@+id/othValue"
+        tools:text="+45,678.00" />
+
+    <EditText
+        android:id="@+id/rthText"
+        android:layout_width="120dp"
+        android:layout_height="wrap_content"
+        android:background="@null"
+        android:ems="10"
+        android:inputType="text"
+        android:text="Market Open:"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/name" />
+
+    <EditText
+        android:id="@+id/othText"
```

Comment:
The positions layout (item_position.xml) is unchanged. Does it also need these changes or is it ok to not include marked closed text?

Reply:
Personally I'd rather see both "At close" and "Market closed" values if that's what you're asking. If you want to remove the titles themselves then sure, but we need to make sure it's clear which values represent what (open/close/at close).